cdap-ui/app/tracker/templates/main.html

```diff
@@ -18,9 +18,42 @@
   <div class="form-group has-feedback tracker-search" ng-keypress="MainController.search($event)">
     <input type="text"
            class="form-control"
-           placeholder="Search datasets"
+           placeholder="Search"
            ng-model="MainController.searchQuery"
-           pattern="^(?![*]).*$">
+           pattern="^(?![*]).*$"
+           cask-focus="searchField">
     <span class="fa fa-search form-control-feedback"></span>
   </div>
+
+  <div class="search-tips">
+    <h3 class="title">
+      Search Tips
+    </h3>
+    <div class="subtitle">
+      Search by keyword: a property, tag, and dataset name
+    </div>
+
+    <div class="tips-table">
+      <table class="table table-responsive">
```

Comment:
This would be a good opportunity for us to see if we could use the table component from react side. Don't think its possible out of the box today but worth seeing what it takes for us to start using more react components like tables and modals in angular side.